app/grandchallenge/cases/views.py

```diff
@@ -88,27 +90,53 @@ class RawImageUploadSessionViewSet(
     def perform_create(self, serializer):
         serializer.save(creator=self.request.user)
 
+    def validate_staged_files(self):
+        upload_session: RawImageUploadSession = self.get_object()
+
+        file_ids = [
+            f.staged_file_id for f in upload_session.rawimagefile_set.all()
+        ]
+
+        if not all(f_id is not None for f_id in file_ids):
```

Comment:
It might be nicer to use
```suggestion
        if any(f_id is None for f_id in file_ids):
```
I  think removing the double negation improves readability a bit.

Reply:
Agreed, thanks!